tools/interactive/interactive_pampa.xml

```diff
@@ -0,0 +1,55 @@
+<tool id="interactive_tool_pampa" tool_type="interactive" name="pampa" version="0.0.1" profile="22.01">
+    <description>Performance Indicators of Marine Protected Areas for the management of coastal ecosystems, resources and their uses</description>
+    <requirements>
+        <container type="docker">ylebras/pampa:latest</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="metashrimps visualisation" requires_domain="True">
+            <port>3838</port>
+            <url>/sample-apps/PAMPA/</url>
+        </entry_point>
+    </entry_points>
+    <command><![CDATA[
+
+        mkdir Results
+        ln -s /srv/shiny-server/sample-apps/PAMPA/Results Results
```

Comment:
I assume this will crash as the ln will complain that a folder with this name already exist?

Reply:
It was crashing but just because I forget the && ;)